src/Modules/ElementUtils/defaultValue.js

```diff
@@ -10,11 +11,11 @@ function hasDefault(element) {
     throw new MissingCacheObjectException(element);
   }
 
-  if (isMember(element)) {
-    return element.content.value.cache.hasDefault;
+  if (!isMember(element) || isUndefined(element.content.value)) {
```

Comment:
I find it is simpler to read without the `!` and `||` and perhaps more logical: If we have a member and the member has a value:

```js
if (isMember(element) && element.content.value) {
  return element.content.value.cache.hasDefault;
}

return element.cache.hasDefault;
```

Feel free to ignore this comment though.

Reply:
I agree, addressed